### PR TITLE
Added missing copyright header

### PR DESCRIPTION
**Pull request by o-kopysov**
Signed-off-by: Oleg Kopysov <o.kopysov@samsung.com>

# Description

Added missing copyright header into unit test file

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [X] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/test/java/com/lpvs/entity/config/WebhookConfigTest.java [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/**
+ * Copyright (c) 2022, Samsung Research. All rights reserved.
```
Comment (MoonkiHong) [inline src/test/java/com/lpvs/entity/config/WebhookConfigTest.java:2]:
```suggestion
 * Copyright (c) 2022, Samsung Electronics Co., Ltd. All rights reserved.
```

**Review by MoonkiHong — COMMENTED** [review]
PTAL. 🙏

File: src/test/java/com/lpvs/entity/config/WebhookConfigTest.java (hunk 1 above)
Comment (o-kopysov) [reply src/test/java/com/lpvs/entity/config/WebhookConfigTest.java]: Thanks for the comment. I've updated all headers.

**Review by MoonkiHong — APPROVED** [review]
+1 - well aligned with my another review comment for https://github.com/Samsung/LPVS/pull/19#pullrequestreview-1074836973.

**Pull request merged**